### PR TITLE
[JENKINS-65928] Job trend page shows previous icons

### PR DESCRIPTION
**Pull request by benebsiny**
See [JENKINS-65928](https://issues.jenkins-ci.org/browse/JENKINS-65928).

**Replace the old icons with the new svg icons**

Screenshot_20210828_144431

**As for the colors of icons in trend graph, because they are created by 3rd party plugin, I'm unable to modified it.**

### Proposed changelog entries

* Replace the old icons with the new SVG icons in the job trend page.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js [hunk 1]
```diff
@@ -33,3 +32,63 @@ function buildTimeTrend_displayBuilds(data) {
...
+	svg1.appendChild(use1);
+
+	const svg2 = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+	svg2.setAttribute('class', 'svg-icon icon-' + iconName + ' icon-sm');
+	svg2.setAttribute('viewBox', "0 0 24 24");
+	const use2 = document.createElementNS('http://www.w3.org/2000/svg', 'use');
+	use2.setAttribute('href', imagesURL + '/build-status/build-status-sprite.svg#' + buildStatus)
+	svg2.appendChild(use2);
+
+	const span = new Element('span', {class: 'build-status-icon__wrapper icon-' + iconName + ' icon-sm', style: 'width: 16px; height: 16px;'}).
```
Comment (timja) [inline core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js:88]: You shouldn’t need to set icon-sm and the style afaik, the css class should handle it?

Comment (benebsiny) [reply core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js:88]: You're right. Thanks for the suggestion!

**Review by timja — APPROVED** [review]
> As for the colors of icons in trend graph, because they are created by 3rd party plugin, I'm unable to modified it.

not actually a plugin but a library bundled in Jenkins, I tracked it down to https://github.com/stapler/stapler-adjunct-timeline

FTR the page being fixed is `http://localhost:8080/job/<job-name>/buildTimeTrend`

The Build History page at http://localhost:8080/view/all/builds is not fixed but can be done in a separate PR

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me as a bugfix. I am not 100% sure it is up for backporting, but I am not against that too

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by timja** [comment]
> Looks good to me as a bugfix. I am not 100% sure it is up for backporting, but I am not against that too

No need to imo

**Pull request merged**